src/element.ts

```diff
@@ -453,27 +438,22 @@ export abstract class Element {
     return size;
   }
 
-  /**
-   * @returns the font size in `pt`.
-   */
+  /** @returns the font size in `pt`. */
   get fontSizeInPoints(): number {
     return Font.convertSizeToPointValue(this.fontSize);
   }
 
-  /**
-   * @returns the font size in `px`.
-   */
+  /** @returns the font size in `px`. */
   get fontSizeInPixels(): number {
     return Font.convertSizeToPixelValue(this.fontSize);
   }
 
-  /**
-   * @returns a CSS font-style string (e.g., 'italic').
-   */
+  /** @returns a CSS font-style string (e.g., 'italic'). */
   get fontStyle(): string {
     return this.fontInfo.style;
   }
 
+  //** Set the font style. */
```

Comment:
@rvilarl I'm going to merge this PR ... but in your next PR can you fix this minor typo. The comment starts with two slashes but should only have one slash.